graphics/src/ColladaLoader.cc

```diff
@@ -2028,10 +2241,10 @@ void ColladaLoader::Implementation::LoadPolylist(
     std::string offset = polylistInputXml->Attribute("offset");
     if (semantic == "VERTEX")
     {
-      unsigned int count = norms.size();
+      unsigned int count = (*norms).size();
```

Comment:
This is where the segfault is happening in CI.  Since norms is an uninitialized shared_ptr, `norms->size()` isn't a valid call at this point.  I suppose it depends on your compiler/system if something kinda bad (data corruption) or really bad (segfault) happens at this point, because it seems to only consistently happen on my jammy system.

Reply:
If fixed this with the latest commit.

Reply:
The remaining test run failure for the Collada Loader seems to be the result of a mismatch between the actual mesh vertex and normal count and the expected counts